src/services/PartsKit.php

```diff
@@ -33,19 +35,7 @@ public static function isRequest(): bool
      */
     public static function getConfig(string $key)
     {
-        // Merge user settings with the defaults
-        $userSettings = Craft::$app->config->getConfigFromFile('viget')['partsKit'] ?? [];
-        $config = array_merge(
-            [
-                'directory' => 'parts-kit',
-                'layout' => '_layouts/app',
-                'volume' => 'partsKit',
-                'theme' => 'light',
-            ],
-            $userSettings
-        );
-
-        return $config[$key] ?? null;
+        return Module::$config['partsKit'][$key] ?? null;
```

Comment:
so for a moment i completely dropped this method and replaced the instances of `self::getConfig` with `Module::$config['partsKit']` -- buuuut we are actually using this method in the twig template to see which layout to extend. figured it was fine to keep, not sure if there's a better way to expose the static var to twig 🤷

Reply:
Yeah agree, makes sense to keep